modules/cnv-troubleshooting-vmware-vm-import.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc
+[id='troubleshooting-vmware-vm-import-{context}']
+= Troubleshooting a VMware virtual machine import
+
+If an imported virtual machine's status is `Import error: (VMware)`, you can check the Conversion Pod log for errors:
+
+. Obtain the Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+
+= Error messages
+
+* Virtual machine event error message: `Readiness probe failed`
++
+Log message: `INFO - have error: ('virt-v2v error: internal error: invalid argument: libvirt domain ‘v2v_migration_vm_1’ is running or paused. It must be shut down in order to perform virt-v2v conversion',)"`
++
+Resolution: Shut down the virtual machine before importing it.
+
+ifeval::[{CNVVersion} == "2.3"]
+= Known issues
```

Comment:
Same comment here that there shouldn't be another level 1 heading in the same module file. But I'm not sure what the appropriate level heading for this one is.